maintainers/scripts/luarocks-packages.csv

```diff
@@ -84,6 +84,7 @@ luasocket,,,,,,
 luasql-sqlite3,,,,,,vyp
 luassert,,,,,,
 luasystem,,,,,,
+luatext,https://github.com/f4z3r/luatext,,,,,
```

Comment:
the package being on luarocks, you dont need `https://luarocks.org/modules/f4z3r/luatext` anymore. NB: The format has changed too and now this veryo column expects a full path towards the rockspec.